xla/service/algorithm_util.cc

```diff
@@ -200,7 +206,7 @@ bool IsSupportedDotAlgorithmOnGpu(
       }
     case PrecisionConfig::ALG_DOT_BF16_BF16_F32_X3:
     case PrecisionConfig::ALG_DOT_BF16_BF16_F32_X6:
-      return (is_cuda_ge_ampere || is_rocm_mi100_and_above) &&
+      return (is_cuda_ge_ampere || is_rocm_mi300_and_above) &&
```

Comment:
Isn't this going to select a different algorithm going forward for mi100 and 250 if you change this? I am not sure this is a good idea without a lot of testing on those cards.